Docs: add Usage Insights documentation (#23982)
grafana/grafana: docs/sources/enterprise/_index.md

[White labeling]({{< relref "white-labeling.md" >}}) allows you to replace the Grafana brand and logo with your own corporate brand and logo. You can also change footer links to point to your custom resources.

> ## Usage Insights

Reviewer: Is "Usage Insights" actually used in the UI? If so, does this text match the UI text?

Author: No it's the generic feature name, I changed it to sentence case.

> [Usage Insights]({{< relref "usage-insights.md" >}}) allows you to understand how your Grafana instance is used. You can see who is looking at a dashboard, how often this dashboard is seen or encounters errors. You'll also be able to discover what are the least and the most used dashboards.

Reviewer: ```suggestion
[Usage Insights]({{< relref "usage-insights.md" >}}) allow you to understand how your Grafana instance is used. You can see who is looking at a dashboard, how often a dashboard is seen, and which dashboards are prone to errors. You'll also be able to discover what are the least and the most used dashboards.
```
